tests/sklearn/test_sklearn_model_export.py

```diff
@@ -698,16 +698,51 @@ def test_sklearn_compatible_with_mlflow_2_4_0(sklearn_knn_model, tmp_path):
 utc_time_created: '2023-07-04 07:19:43.561797'
 """
     )
+    tmp_path.joinpath("python_env.yaml").write_text(
+        """
+python: 3.8.16
+build_dependencies:
+   - pip==23.1.2
+   - setuptools==56.0.0
+   - wheel==0.40.0
+dependencies:
+   - -r requirements.txt    
+"""
+    )
+    tmp_path.joinpath("requirements.txt").write_text(
+        """
+mlflow==2.4
```

Comment:
```suggestion
mlflow==2.4.x
```

I think we should specify the micro version.